libmscore/scorefile.cpp

```diff
@@ -512,6 +512,15 @@ bool MasterScore::saveFile(bool generateBackup)
       return true;
       }
 
+//---------------------------------------------------------
+//    isFileExisting
+//---------------------------------------------------------
+
+bool MasterScore::isFileExisting() const
```

Comment:
There is a `_created` flag, probably it should be used

Reply:
yep, didn't notice it